.drone.yml

```diff
@@ -1,32 +1,63 @@
+---
 kind: pipeline
-type: docker
+type: kubernetes
 name: default
 
-
 steps:
   - name: test
-    image: golang:1.13
-    commands:
-      - go test ./cmd/... ./internal/...
-      - go vet ./cmd/... ./internal/...
-  - name: lint
-    image: cytopia/golint
+    image: golangci/golangci-lint:v1.30.0-alpine
     commands:
-      - golint -set_exit_status ./cmd/... ./internal/...
-  - name: build
-    image: golang:1.13
+      - apk update
+      - apk add make
+      - make test
+
+  - name: snyk
+    image: snyk/snyk:golang-1.14
+    environment:
+      SNYK_TOKEN:
+        from_secret: snyk_token
     commands:
-      - go build -o build/drone-helm cmd/drone-helm/main.go
+      - snyk monitor
+    when:
+      event: 
+        - push
+
   - name: publish_linux_amd64
     image: plugins/docker
     settings:
-      auto_tag: true
+      registry: quay.io
+      repo: quay.io/mongodb/drone-helm
+      username:
+        from_secret: docker_username
+      password:
+        from_secret: docker_password
+      tags:
+        - git-${DRONE_COMMIT_SHA:0:7}
+        - latest
```

Comment:
We shouldn't publish `latest` so that users are forced to pick a version.

Reply:
done